godtools/Views/TractElements/Cards.swift

```diff
@@ -66,6 +66,11 @@ class Cards: BaseTractElement {
     override func setupView(properties: Dictionary<String, Any>) {
         self.frame = buildFrame()
         self.backgroundColor = .clear
+        
+        let image = UIImage(named: self.backgroundImagePath)
```

Comment:
Yes we have to, I didn't do that because the options were missing on the element (is still as WIP), but will make something and add suggestions to the XML file